packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -16,7 +16,11 @@ import { store } from '../../store';
 // This is used to avoid rendering the block list if the sizes change.
 let MemoizedBlockList;
 
-function AutoBlockPreview( { viewportWidth, __experimentalPadding } ) {
+function AutoBlockPreview( {
+	viewportWidth,
+	__experimentalPadding,
+	__experimentalIsRootContainer,
```

Comment:
I'm hesitant about this prop, it's very weird to be honest. it's a hack because the issue here is not really the real issue here, the issue is the added div with its own layout. So while I'm ok with this PR, I was wondering if we can borrow the `useBlockPreview` from the other PR?

Reply:
Me too. I was kind of expecting to close it to be honest 😄 

>I was wondering if we can borrow the useBlockPreview from the other PR?

I'll try this out.

Reply:
> I was wondering if we can borrow the useBlockPreview from the other PR?

Thanks for exploring that! I quite like the `useBlockPreview` we came up with in that other PR, so it'd be cool if we can include it here 🙂

Reply:
@andrewserong would you be interested in taking over this or maybe better creating a PR with `useBlockPreview`? I tried this a bit and it seems most of the code needed had been implemented by you in: https://github.com/WordPress/gutenberg/pull/35863.

Reply:
@ntsekouras sure, I'm happy either take this over or create another PR. The only thing is I won't be able to start on it until later this week in case there's urgency in getting this in sooner. If not, then I can pick it up 🙂

Reply:
That sounds great, thanks! 💯 This is a bug, so there is time to fix it and include it in 5.9.

Reply:
I'm wrapping up for the week, but I've started on a fix and pushed a draft PR in https://github.com/WordPress/gutenberg/pull/36431 that appears to work okay so far — I haven't had a chance to test it properly or see if it can be streamlined a bit, so I've left it as a draft PR for the moment, and will continue on with it on Monday.

Reply:
That's great Andrew! Thanks! I'll close this one and we can use your new PR.